client/jetpack-onboarding/steps/site-type.jsx

```diff
@@ -20,6 +20,9 @@ import { saveJetpackOnboardingSettings } from 'state/jetpack-onboarding/actions'
 
 class JetpackOnboardingSiteTypeStep extends React.PureComponent {
 	handleSiteTypeSelection = siteType => () => {
+		this.props.recordJpoEvent( 'calypso_jpo_site_type_clicked', {
+			siteType: siteType,
```

Comment:
```
siteType: siteType,
```

can be shortened to just:

```
siteType,
```

(see https://ariya.io/2013/02/es6-and-object-literal-property-value-shorthand)

Reply:
yes! Slipped my mind 👍